packages/core/core/src/services/content-source-maps.ts

```diff
@@ -0,0 +1,201 @@
+import { vercelStegaCombine } from '@vercel/stega';
+import type { Core, Struct } from '@strapi/types';
+
+const ENCODABLE_TYPES = [
+  'string',
+  'text',
+  'richtext',
+  'uid',
+  'enumeration',
+  'email',
+  'date',
+  'datetime',
+  'time',
+  'timestamp',
+];
+
+const EXCLUDED_FIELDS = [
+  'id',
+  'documentId',
+  'locale',
+  'localizations',
+  'created_by',
+  'updated_by',
+  'created_at',
+  'updated_at',
+  'publishedAt',
+];
```

Comment:
can we get this list of fields from somewhere else? Are this internal strapi fields? Or the ones not visible in the content manager?

Asking cause its tricky to maintain lists like this over time, and we often forget to update them.

Reply:
@innerdvations agrees with you :) https://github.com/strapi/strapi/pull/20999/files#diff-fa1aa6b92551dda5664e962046435170c28dac858d50e033eebb60b62035d1d9R5

Reply:
It's a bit too much of a side quest for this PR imo so I added a tech debt entry to do this: https://www.notion.so/strapi/6619863711f74711a4fa4db73e16c517?v=1268f3598074807eb62b000c96893028&p=25a8f359807480c98c15c7cda8cf79c5&pm=s